lib/symbolize/mongoid.rb

```diff
@@ -64,7 +64,7 @@ def symbolize(*attr_names) # rubocop:disable Metrics/AbcSize
         default_option = configuration[:default]
 
         field_type     = configuration[:type] || String
-        enum           = [true, false] if [Boolean, ::Boolean].include?(field_type)
+        enum           = [true, false] if [Boolean].include?(field_type)
```

Comment:
:information_source: Since it's only a list of one thing:
```suggestion
        enum           = [true, false] if field_type == Boolean
```